k4e-agent-sti.sh

```diff
@@ -20,25 +20,32 @@ usage: $0 options
 This script will create an image file and ISO to be served for download.
 OPTIONS:
    -h      Show this message
+   -a      Image server address
    -i      Image name (required)
    -o      Operator's HTTP API (required)
    -v      Verbose
 EOF
 }
 
-while getopts "h:i:o:v" option; do
+while getopts "h:a:i:o:v" option; do
     case "${option}"
     in
         h)
             usage
             exit 0
             ;;
+        a) IMAGE_SERVER_ADDRESS=${OPTARG};;
         i) IMAGE_NAME=${OPTARG};;
         o) HTTP_API=${OPTARG};;
         v) VERBOSE=1;;
     esac
 done
 
+if [[ -z $IMAGE_SERVER_ADDRESS ]]; then
+    echo "ERROR: server address is required"
```

Comment:
if IMAGE_SERVER_ADDRESS isn't provided, let's check if `host $(hostname)` fails and only then let's fail the script.
else, let's set
`IMAGE_SERVER_ADDRESS=$(hostname)`

Reply:
"host" is not a default package that is installed and we are only validating parameters, I'm not sure it's correct to install packages here...

Reply:
done